File: And_2.py
weather = "sunny"
temperature = "cold"

if weather == "raining" and temperature == "cold":
    print("Take an umbrella and a warm jacket")
elif weather == "raining" and temperature == "warm": # with And both checks must be ==
    print("Take an umbrella and a t-shirt")
elif weather == "sunny" and temperature == "cold":
    print("Take sunglasses and a warm jacket")
elif weather == "sunny" and temperature == "warm":
    print("Take an umbrella and a t-shirt")
else:
    print("Stay home")

File: And_2.js
var weather = "sunny"
var temperature = "warm"

if (weather == "raining" && temperature == "warm") {
    console.log("Take an umbrella and sunglasses");
} else if ( weather == "raining" && temperature == "cold") {
    console.log("Take an umbrella and a warm jacket");
} else if ( weather == "sunny" && temperature == "warm") {
    console.log("Take sunglasses and a t-shirt");
} else if ( weather == "sunny" && temperature == "cold") {
    console.log("Take sunglasses and a warm jacket");
} else {
    console.log("Stay home!")
}